### ISSUE TITLE
Missing form label on Email field - Accessibility

### ISSUE THREAD

**Issue by phucduong86** [shiffman/shiffman.net#70]
Should be a quick fix with "aria-label"

<img width="427" alt="home daniel shiffman 2018-11-06 21-12-15" src="https://user-images.githubusercontent.com/21692768/48106248-b3441780-e208-11e8-9a6a-1b9c7d1f8fea.png">

**Comment by phucduong86** [comment]
Issue has been resolved.
![image](https://user-images.githubusercontent.com/21692768/55327894-1c972200-5459-11e9-8de7-42c5c4441e7b.png)
